ch05.adoc

```diff
@@ -25,7 +25,11 @@ If the longitude, latitude, vertical or time coordinate is multi-valued, varies
 
 If an **`axis`** attribute is attached to an auxiliary coordinate variable, it can be used by applications in the same way the **`axis`** attribute attached to a coordinate variable is used. However, it is not permissible for a data variable to have both a coordinate variable and an auxiliary coordinate variable, or more than one of either type of variable, having an **`axis`** attribute with any given value e.g. there must be no more than one **`axis`** attribute for **`X`** for any data variable. Note that if the **`axis`** attribute is not specified for an auxiliary coordinate variable, it may still be possible to determine if it is a spatiotemporal dimension from its own units or standard_name, or from the units and standard_name of the coordinate variable corresponding to its dimensions (see <<coordinate-types>>). For instance, auxiliary coordinate variables which lie on the horizontal surface can be identified as such by their dimensions being horizontal. Horizontal dimensions are those whose coordinate variables have an **`axis`** attribute of **`X`** or **`Y`**, or a **`units`** attribute indicating latitude and longitude.
 
-If the coordinate variables for a horizontal grid are not longitude and latitude, it is recommended that they be supplied __in addition__ to the required coordinates. For example, the Cartesian coordinates of a map projection should be supplied as coordinate variables in addition to the required two-dimensional latitude and longitude variables that are identified via the **`coordinates`** attribute. The use of the **`axis`** attribute with values **`X`** and **`Y`** is recommended for the coordinate variables (see <<coordinate-types>>).
+To geo-reference data horizontally with respect to the Earth (or other body), a __grid mapping variable__ may be provided by the data variable, using the **`grid_mapping`** attribute.
```

Comment:
As much as I find the idea of using CF on other planetary bodies cool, we may want to tackle that - if a need is identified - in a separate issue. The phrasing "(or other body)" doesn't really align with terminology used elsewhere in the document, where we talk about **geo**location, etc. If we're using a planetocentric or planetographic coordinate system a lot of the assumptions that you can use in Earthbound CRS don't apply. I don't believe this is insurmountable but would argue that it is deserving of further attention outside of this issue if we want to address it.

Reply:
agreed.  This was an issue for another activity i worked on recently, but not relevant here; I think I inserted it by habt, not design